perf-system/generator/generator.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+"""
+Generate requests
+"""
+
+import pandas as pd  # type: ignore
+
+# pylint: disable=import-error
+import fastparquet as fp  # type: ignore
+
+REQUEST_CONTENT_TYPE = "content-type: application/json"
+REQUEST_LENGTH_TEXT = "content-length: "
+
+df = pd.DataFrame(columns=["messageID", "request"])
+
+
+def fill_df(
+    host: str, req_path: str, req_type: str, req_verb: str, req_iters: int, data: str
+):
+    """
+    Creates a dataframe with the data
+    required for the requests
+    """
+    # entering the private file paths as metadata in the start of parquet
+
+    print("Starting generation of requests")

```

Comment:
We use loguru elsewhere in the infra and tests, I would suggest doing the same. It does a bunch of nice things for free: timestamps, colours, levels etc.

Reply:
I don't think it makes sense to use any logging in this component since I replaced `fill_df`. Although, I will replace my generator component logs with loguru,